src/main/java/helper/RequestValidator.java

```diff
@@ -12,25 +13,25 @@
 
 	private List<String> getMissingFields(Map<String, Object> data, String[] required) {
```

Comment:
The value type here is not relevant or used.
Does this need to be a Map or can it be a Set?

If it has to be a Map, you should use `Map<String, ?>` instead of `Map<String, Object>` to remove restrictions on the callers.

Reply:
Changed to`Set<String>`.